packages/expo-modules-autolinking/src/reactNativeConfig/reactNativeConfig.ts

```diff
@@ -69,13 +69,24 @@ export async function createReactNativeConfigAsync({
  */
 export async function findDependencyRootsAsync(
   projectRoot: string,
-  searchPaths: string[]
+  searchPaths: string[],
+  platform: SupportedPlatform
 ): Promise<Record<string, string>> {
   const packageJson = JSON.parse(await fs.readFile(path.join(projectRoot, 'package.json'), 'utf8'));
   const dependencies = [
     ...Object.keys(packageJson.dependencies ?? {}),
     ...Object.keys(packageJson.devDependencies ?? {}),
   ];
+  const shouldAutolinkEdgeToEdge =
+    platform === 'android' &&
+    getExpoVersion(packageJson) >= 53 &&
+    !dependencies.includes('react-native-edge-to-edge');
```

Comment:
does this lookup happen before a step where we filter out modules based on config, eg: if they configure react-native.config.js to skip autolinking this module?